serfclient/client.py

```diff
@@ -88,3 +88,10 @@ def close(self):
         if self.connection:
             self.connection.close()
             self.connection = None
+
+    def tag(self, tag, value):
```

Comment:
Is it worth calling this `set_tag` to be explicit?

Reply:
I tried to keep the function name as close to how the Serf RPC & how Serf CLI calls it

Reply:
Yea. I will rename this into `set_tag` and add new function called `delete_tag`